SampleApplications/Samples/GDS/ServerCommon/GlobalDiscoveryServerConfiguration.cs

```diff
@@ -81,7 +81,7 @@ private void Initialize()
         public StringCollection KnownHostNames { get; set; }
 
         [DataMember(Order = 6)]
-        public uint ShutdownDelay { get; set; }
+        public string DatabaseStorePath { get; set; }
         #endregion
```

Comment:
add databaseclassname

Reply:
future

---
In reply to: [155163752](https://github.com/OPCFoundation/UA-.NETStandard/pull/284#discussion_r155163752) [](ancestors = 155163752)